package-lock.json

```diff
@@ -0,0 +1,7123 @@
+{
```

Comment:
Hi, @kalvinarts 
could you please remove `package-lock.json` file.

Reply:
Done